.changeset/tired-places-cough.md

```diff
@@ -0,0 +1,27 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+---
+---
+"section": feature
```

Comment:
Feature is technically correct but I think we're still putting all SharedTree changes in their own section?
```suggestion
"section": tree
```

Reply:
yes, this was a mistake.